src/Input/Silk.NET.Input.Common/InputWindowExtensions.cs

```diff
@@ -0,0 +1,28 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using Silk.NET.Windowing.Common;
+
+namespace Silk.NET.Input.Common
+{
+    public static class InputWindowExtensions
+    {
+        public static void RegisterInputPlatform(IInputPlatform platform)
+        {
+            throw new NotImplementedException();
+        }
+
+        public static void DeregisterInputPlatform(IInputPlatform platform)
```

Comment:
I was yesterday years old when I discovered that `deregister` isn't actually a word, but `unregister` is. Can you change this to `UnregisterInputPlatform`, and reflect the changes in the proposal please?